src/AppInstallerCLICore/Workflows/RepairFlow.cpp

```diff
@@ -354,28 +429,56 @@ namespace AppInstaller::CLI::Workflow
                 GenerateRepairString;
         }
         break;
+        // MSI based installers, for installed package all gets mapped to msi extension.
         case InstallerTypeEnum::Msi:
         case InstallerTypeEnum::Wix:
         {
             context <<
                 SetProductCodesInContext;
         }
         break;
+        // MSIX based installers, msix, msstore.
         case InstallerTypeEnum::Msix:
+        case InstallerTypeEnum::MSStore:
         {
             context <<
                 SetPackageFamilyNamesInContext;
         }
         break;
-        case InstallerTypeEnum::MSStore:
-            break;
         case InstallerTypeEnum::Portable:
         default:
             THROW_HR(HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED));
         }
     }
 
     void RepairMsixPackage(Execution::Context& context)
+    {

```

Comment:
this method is not needed, as comments above

Reply:
It looks beneficial for removing the installer mapping dependency, but I'm willing to adjust if it proves necessary